src/Octoshift/GithubClient.cs

```diff
@@ -115,6 +115,11 @@ public virtual async IAsyncEnumerable<JToken> GetAllAsync(string url)
         {
             url = url?.Replace(" ", "%20");
 
+            if (httpMethod == HttpMethod.Post && url is not null && url.Trim().EndsWith("/v3/graphql", StringComparison.OrdinalIgnoreCase))
+            {
+                url = url.Replace("/v3/graphql", "/graphql", StringComparison.OrdinalIgnoreCase);
+            }
```

Comment:
this turned out not as ugly as I was thinking. This ain't so bad!

Reply:
Yeah I guess this is the easiest approach to put the logic in the client instead of the `GithubApi` 😃